posts/2024-08-07-Project-goals.md

```diff
@@ -0,0 +1,89 @@
+
+## Project goals: a "front door" for Rust
+
+To me, the most exciting thing about the Project Goals program has been seeing the [goals][1] [coming][2] [from][3] [outside][4] the existing Rust maintainers. My hope is that the Project Goal process can supplement RFCs as an effective "front door" for the project, offering people who have the resources and skill to drive changes a way to float that idea and get feedback from the RUst teams *before* they begin to work on it.
+
+[1]: https://rust-lang.github.io/rust-project-goals/2024h2/cargo-semver-checks.html
+[2]: https://rust-lang.github.io/rust-project-goals/2024h2/Rust-for-SciComp.html
+[3]: https://rust-lang.github.io/rust-project-goals/2024h2/user-wide-cache.html
+[4]: https://rust-lang.github.io/rust-project-goals/2024h2/std-verification.html
+
+Project Goals will help ensure the sustainability of the Rust open source community. In the past, it was difficult to tell when starting work on a project whether it would be well-received by the Rust maintainers. This was an obstacle for those who would like to fund efforts to improve Rust, as people only like to fund work if they have some reasonable confidence it will succeed. Project goals are a way for project maintainers to "bless" a particular project and indicate their belief that it will be helpful to Rust. The Rust Foundation is using project goals as one of their criteria when considering [fellowship applications](https://foundation.rust-lang.org/grants/fellowships/), for example, and I expect over time other grant programs will do the same. But project goals are useful for others, too: having an approved project goal can help someone convince their employer to give them time to work on Rust open source efforts, for example, or give contractors the confidence they need to ensure their customer they'll be able to get the work done.
+
+## Follow progress at whatever level of detail is best with you
+
+As the year progresses, you can follow along with the progress on Rust project goals just by looking for our regular blog posts. If you'd like to see more detail, then subscribe the various [tracking issues for the Project Goals that interest you][m], or monitor the [#project-goals channel](https://rust-lang.zulipchat.com/#narrow/stream/435869-project-goals) on the [rust-lang Zulip](https://rust-lang.zulipchat.com). See you there!
```

Comment:
Not sure if intentional (I guess you may want to end with this paragraph), but there is the "How to track progress" section above too which overlaps with this one.